Configuration.cs

```diff
@@ -0,0 +1,12 @@
+namespace DotNetMaui;
+
+public static class Configuration
+{
+
+    public const string SentryDsn = "https://ec46b4c5e07cd8e976033a8a1e63cb01@o87286.ingest.us.sentry.io/4509153190739968";
```

Comment:
Could u add a comment over the DSN to indicate which org/project it belongs to?
Would be helpful if we need to swap between sentry-sdks and the demo org for example

Reply:
also we probably want to use sandbox mirror https://sentry.slack.com/archives/C04MF17ELB0/p1745342575418889

Reply:
How are you planning to release this?  Could use CI to properly inject the secret var?

Reply:
in the long run that would be ideal but until we have a functioning CI workflow that can 1) build it and 2) upload the release to github it's fine to hardcode i think

Reply:
@aritchie this DSN will ensure all events also go into sandbox.sentry.io: https://ec46b4c5e07cd8e976033a8a1e63cb01@sandbox-mirror.sentry.gg/1

Reply:
Switched to the updated DSN